src/librustdoc/html/static/rustdoc.css

```diff
@@ -163,11 +163,17 @@ nav.sub {
 }
 
 .sidebar {
+ 	background-color: #F1F1F1;
```

Comment:
color changes shouldn't be in this file but in `src/librustdoc/html/static/styles/main.css`.